### DISCUSSION TITLE
TRYING TO LOAD DWSIM ON TO A MAC

### DISCUSSION THREAD

**Discussion by jshertok** [DanWBR/dwsim#368] [General]
Hello --

I'm trying to load  DWSIM on to a Mac Mini, which has the latest Mac OS.

I get message, "DWSIM cannot be loaded because Apple cannot check it for malicious software.  The software need to be updated."

What is the problem?

Regards,
Joel Shertok

**Comment by DanWBR** [comment]
@jshertok hold Alt while opening it.

**Comment by jshertok** [comment]
Hello --

I tried this, but the same issue..

I'll need to call Apple Help on Monday... this may be an Apple security
issue.

Thanks,
Joel

**Comment by DanWBR** [comment]
@jshertok http://www.phy.ohio.edu/~hadizade/blog_files/tag-how-to-allow-apps-from-anywhere-in-macos-gatekeeper-.html

**Comment by DanWBR** [comment]
@jshertok DWSIM for Mac is signed with my Apple developer certificate. This issue happens because I didn't notarize the app, but you can safely run it.

**Comment by jshertok** [comment]
Hello, Dan ---

Thanks for your response....

My issue is that when I try to download DWSIM on my Mac, it is
immediately blocked...

I suspect that I will need to call Apple support on Monday (or download
DWSIM on my old, clunky Lenovo PC).

Regards,
Joel
